### PR TITLE
Add transport-analysis as MDAKit

### PR DESCRIPTION
**Pull request by hmacdope**
Add transport-analysis as MDAKit, a tool for analysing transport properties including diffusion coefficents.

### CODE REVIEW

**Comment by hmacdope** [comment]
@MDAnalysis/mdakits-reviewers, ready for review.

File: mdakits/transport-analysis/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+keywords:
+  - transport properties
+  - molecular dynamics
+license: GPL-2.0
+project_home: https://github.com/MDAnalysis/transport-analysis
+documentation_home: https://transport-analysis.readthedocs.io
+documentation_type: UserGuide + API
+
+# Optional entries
+install:
```
Comment (IAlibay) [inline mdakits/transport-analysis/metadata.yaml:20]: Any chance you c might be able to make a release before adding to the registry?

**Comment by IAlibay** [comment]
I'm not going to block over it since everything else is fine, but a release would be significantly better in terms of a) not having to come back and fix this registry entry later, b) making sure users can access the code properly.

**Comment by hmacdope** [comment]
@IAlibay done, 0.1.0 initial release done.

**Comment by IAlibay** [comment]
Can we switch the install to a pypi one?

**Comment by hmacdope** [comment]
@IAlibay should be good now.

**Review by IAlibay — APPROVED** [review]
cheers!

**Pull request merged**